packages/docs/docs/api/cli.md

```diff
@@ -1,52 +1,74 @@
+---
+sidebarDepth: 3
+---
+
 # Command-line Interface
 
+There are currently four cli commands in VuePress: [build](#build), [dev](#dev), [eject](#eject) and [info](#info).
+
+If they are not enough for you, you can also create [your own commands](#more-commands).
+
 ## Usage
 
 ```bash
-vuepress <command> targetDir [options]
```

Comment:
Why removing targetDir?

Reply:
`vuepress info` command does not require `targetDir `

Reply:
Then the other commands miss `targetDir`

Reply:
I think that's the difference between `vuepress --help` and `vuepress dev --help`?

Reply:
Well, some people won't realize that they have to pass `targetDir ` at first glance. In this case, how about rewriting `Usage` in each section?

Reply:
After reviewing it again, I think I'm not making too much improvement 😅Really sorry about that, will do more research next time. Does it look a bit better now?

Reply:
LGTM!